enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -639,6 +640,15 @@ func (c *ociContainer) createSpec(ctx context.Context, cmd *repb.Command) (*spec
 	if err != nil {
 		return nil, fmt.Errorf("get container user: %w", err)
 	}
+
+	cpuSpecs := &specs.LinuxCPU{}
+	if *cpuLimit != 0 {
+		cpuSpecs = &specs.LinuxCPU{
+			Quota:  pointer(int64(*cpuLimit * 100_000)),
+			Period: pointer(uint64(100_000)),
```

Comment:
a little comment about what units we're using here would be helpful for the reader

Reply:
fixed